### ISSUE TITLE
linux input a russian letters

### ISSUE THREAD

**Issue by Dolfost** [efroemling/ballistica#259]
I installed the game on the Linux machine Ubuntu (latest version) Bomb Squad from Ubuntu Store and can not write Russian letters in the chat. How to repair it? 

In SAMP I was helped to write before calling the program (wine) "LANG = ru_RU.utf8 "

**Comment by Dliwk** [comment]
Game shows empty cells instead of Russian letters or what?

**Comment by Dolfost** [comment]
> Game shows empty cells instead of Russian letters or what?

@Dliwk no, the pointer doesn't even move when I try to enter something in Cyrillic

**Comment by Dliwk** [comment]
And have you tried running game from command line with configured LANG environment variable (as in case you mentioned)?

**Comment by Dolfost** [comment]
> And have you tried running game from command line with configured LANG environment variable (as in case you mentioned)?

No, i dont know how launch bombsquad from terminal. I run it from the desktop. By the way, if you enter the game when you are in the Cyrillic layout, you need to change the buttons in the settings again. And if you go on the English cover then there is no such thing.

**Comment by Dliwk** [comment]
> By the way, if you enter the game when you are in the Cyrillic layout, you need to change the buttons in the settings again.

Looks like #38, perhaps this is an SDL bug.

> No, i dont know how launch bombsquad from terminal.

You can somehow get location of executable, e.g. try click right mouse button BombSquad icon (then press edit/read/etc) and search there. Then paste this path in console.

By the way, you can set LANG variable globally: you can try to add this line somewhere (`/etc/default/locale`, `/etc/locale.conf`, or as a last resort `/etc/environment`), after that, it will be logical to reboot.
```bash
LANG=ru_RU.UTF-8
```

**Comment by Dliwk** [comment]
And could you show output of `locale` command?

**Comment by Dolfost** [comment]
> And could you show output of `locale` command?

 ~ locale   
LANG=en_GB.UTF-8
LANGUAGE=en_GB:en
LC_CTYPE="en_GB.UTF-8"
LC_NUMERIC=uk_UA.UTF-8
LC_TIME=uk_UA.UTF-8
LC_COLLATE="en_GB.UTF-8"
LC_MONETARY=uk_UA.UTF-8
LC_MESSAGES="en_GB.UTF-8"
LC_PAPER=uk_UA.UTF-8
LC_NAME=uk_UA.UTF-8
LC_ADDRESS=uk_UA.UTF-8
LC_TELEPHONE=uk_UA.UTF-8
LC_MEASUREMENT=uk_UA.UTF-8
LC_IDENTIFICATION=uk_UA.UTF-8
LC_ALL=
➜  ~

**Comment by Dolfost** [comment]
> ```shell
> LANG=ru_RU.UTF-8
> ```
If I change LANG to Russian, the computer language will change to Russian. In SAMP it was the same, when the Russian language on the computer, everything worked fine. But I do not want my PC to be in Russian.

**Comment by Dolfost** [comment]
#  File generated by update-locale
LANG=en_US.UTF-8
LANG=ru_RU.UTF-8
LC_NUMERIC="uk_UA.UTF-8"
LC_TIME="uk_UA.UTF-8"
LC_MONETARY="uk_UA.UTF-8"
LC_PAPER="uk_UA.UTF-8"
LC_NAME="uk_UA.UTF-8"
LC_ADDRESS="uk_UA.UTF-8"
LC_TELEPHONE="uk_UA.UTF-8"
LC_MEASUREMENT="uk_UA.UTF-8"
LC_IDENTIFICATION="uk_UA.UTF-8"
#LANGUAGE="uk:en"
lockscreen on russian, pc on english. bombsquad input dont work


#  File generated by update-locale
#  File generated by update-locale
# LANG=en_US.UTF-8
LANG=ru_RU.UTF-8
LC_NUMERIC="uk_UA.UTF-8"
LC_TIME="uk_UA.UTF-8"
LC_MONETARY="uk_UA.UTF-8"
LC_PAPER="uk_UA.UTF-8"
LC_NAME="uk_UA.UTF-8"
LC_ADDRESS="uk_UA.UTF-8"
LC_TELEPHONE="uk_UA.UTF-8"
LC_MEASUREMENT="uk_UA.UTF-8"
LC_IDENTIFICATION="uk_UA.UTF-8"
#LANGUAGE="uk:en"
~
Lockscreen on rus, pc on eng, input in bombsquad dont work...

**Comment by Dliwk** [comment]
So, have you got executable location?..

**Comment by Loup-Garou911XD** [comment]
Closing as outdated and stale
